versions.json

```diff
@@ -13,7 +13,7 @@
     },
     "spdk": {
         "repo": "https://github.com/longhorn/spdk.git",
-        "commit": "3028cc58ac215768d3b89298d63d38564d7d6f95",
+        "commit": "be2fba328b9e2d973ad53824ecbfd319c09e9fec",
         "tag": "v24.09.0+4",
```

Comment:
Please create a tag `v24.09.0+4` in longhorn/spdk and bump the tag to `v24.09.0+4`.

Reply:
> Do we have a procedure how to create tag?

1. Create a tag `v24.09.0+5` from the commit `be2fba328b9e2d973ad53824ecbfd319c09e9fec` in your local repo
2. Push the tag to upstream repo